include/seqan3/range/view/repeat_n.hpp

```diff
@@ -0,0 +1,92 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
+ * \brief Provides seqan3::view::char_to.
+ */
+
+#pragma once
+
+#include <seqan3/range/view/repeat.hpp>
+#include <seqan3/range/view/take_exactly.hpp>
+
+namespace seqan3::detail
+{
+
+/*!\brief The underlying type of seqan3::view::trim.
+ * \ingroup view
+ *
+ * Under the hood this delegates to view::take_until.
+ */
+struct repeat_n_fn
+{
+    /*!\brief todo
+     * \tparam irng_t The type of the range being processed. See seqan3::view::trim for requirements.
+     * \param irange The range being processed.
+     * \param threshold The minimum quality as a phred score [integral type].
+     */
+    template <typename value_t>
+    constexpr auto operator()(value_t && value, size_t const count) const
+    {
+        static_assert(std::CopyConstructible<value_t>, "The value passed to repeat_n must be copy constructible.");
```

Comment:
I think this is included somewhere in repeat.hpp or take_exaclty.hpp because the compiler does not complain :)

Reply:
I have in mind that we do not want these chain-dependencies: If you change code and for some reason want to remove the header that itself includes <concepts>, it is not visible that this header is still required. Additionally, if we rely on compiler complaints, we create situations, where the order of include statements matters. Isn't the purpose of our header tests to prevent that?

Reply:
I included the header.